receiver/splunkenterprisereceiver/metadata.yaml

```
@@ -45,6 +45,12 @@ attributes:
   splunk.feature.health:
     description: The Health (in color form) of a Splunk Feature from the Splunk Health Introspection Endpoint
     type: string
+  splunk.searchable.restart:
+    description: String that indicates if a searchable rolling restart/upgrade in progress.t
```

```suggestion
    description: String that indicates if a searchable rolling restart/upgrade is in progress.
```
What is the value of that string? This may create high cardinality.

---

Looks like you can use booleans instead: https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/mdatagen/metadata-schema.yaml#L80

---

yeah that's right I switched it to boolean